custom_components/unifi_insights/api/network/models/lag.py

```
@@ -0,0 +1,83 @@
+"""Link Aggregation Group (LAG) models for UniFi Network API."""
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any
+
+from pydantic import BaseModel, Field
+
+
+class LagType(str, Enum):
```

Ruff (0.16.0)

[warning] Class LagType inherits from both `str` and `enum.Enum`

Inherit from `enum.StrEnum`

(UP042)